synapse/push/mailer.py

```diff
@@ -291,6 +291,13 @@ async def send_email(self, email_address, subject, template_vars):
         if raw_to == "":
             raise RuntimeError("Invalid 'to' address")
 
+        template_vars = {
+            "app_name": self.app_name,
+            "server_name": self.hs.config.server.server_name,
+        }
```

Comment:
Were these passed manually in some cases that we should now remove from the callers?

Reply:
Goodpointwellmade.